CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [Backup/exports] Retry reading a block during delta backup, vhd, ova and vmdk exports (PR [#6763](https://github.com/vatesfr/xen-orchestra/pull/6763))
```

Comment:
```suggestion
- [Backup, Disk export] Automatically retry downloading a block when NBD is enabled (PR [#6763](https://github.com/vatesfr/xen-orchestra/pull/6763))
```

Reply:
not the xva backup or exports

Reply:
Right, it's for the delta backup, but XVAs are not used for disk exports.

Reply:
yes, but disks exports are used for ova exports under the hood

What about 
- [Backup/exports] Retry on failure reading a data block  during Delta Backup, Continuous Replication, disk  and Ova export when NBD is enabled (PR [#6763](https://github.com/vatesfr/xen-orchestra/pull/6763))